bin/mozilla/aa.pl

```diff
@@ -651,7 +651,7 @@ sub form_header {
 
     $form->{onhold} = ( $form->{onhold} ) ? "checked" : "";
 
-    $form->header;
+    $form->header(0, 0, $locale);
```

Comment:
why do we now need to pass 3 params? And also what is 0,0 for?
What does that mean for the proxy or ledger to phoenix mappers?

Reply:
I added the $locale param to the header method. 

`my ( $self, $endsession, $nocookie, $locale ) = @_;
`

0 are the default values. I guess I could also add $locale as first param. Would you prefer that?

Reply:
but then there are 4 params not only 3? 
I prefere readable code, so yeah, if you could pass the correct variable isntead of a 0 would be good I guess.

Reply:
well yes 4 params but first one is $self which means this, so only 3 real params

Reply:
@phkroll Switching the params won't be possible, since then we would have to pass the $locale everywhere the header() method is used. That would mean that we have to pass the $locale variable to all methods that use the header method and so on. 

Also passing the 0s is no problem since 0 is the default value. Before no params where passed (so 0). Where params where passed 0 was also used as default value.